lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -1052,6 +1116,19 @@ def save(self, filename, delimiter=";", sampling_width=None):
         """
         export_kymotrackgroup_to_csv(filename, self, delimiter, sampling_width)
 
+    def _tracks_by_kymo(self):
+        """Find tracks for each `Kymo` in the group.
+
+        Returns
+        -------
+        dict of KymoTrackGroup
+            returns a dictionary where the keys are Kymos which the associated tracks
+        """
+        return {
+            kymo: KymoTrackGroup([track for track in self if track._kymo == kymo])
+            for kymo in self._kymos
+        }
```

Comment:
Considering what will be needed for supporting the refinement functions and that you only ever use `.values()` from the returned `dict` I think it'd be better to just return a `list` here.

For the refinement we'll need to also return the indices that the tracks were originally at so we can reconstruct the group in the same order. So leaving it this way you'd return something like `{kymo: (KymoTrackGroup, list)}` which is just kinda unnecessarily messy